packages/tanstack-start/src/utils/errors.ts

```diff
@@ -24,3 +24,9 @@ To fix this, make sure you have the \`clerkHandler()\` configure in you SSR entr
 
 For more info, check out the docs: https://github.com/clerk/javascript/tree/main/packages/tanstack-start#setup-clerkhandler-in-the-ssr-entrypoint,
     `);
+
+export const warnPackageRenaming = () => {
+  console.warn(
+    '[@clerk/tanstack-start] Warning: This package will be renamed to @clerk/tanstack-react-start in the next minor version. Please stay tuned for the release to update your imports accordingly.',
```

Comment:
I don't like that this is not actionable for a user. Here's what I had in mind:

1. Rename the package locally and bump the minor
2. Publish this new package locally so that it exists on npm
3. Add this warning here with actionable steps for people to migrate (this also includes changing the README of this package
4. Publish it
5. Rename the package in this repo here

This way people can immediately migrate without having to wait

Reply:
Thanks for sharing your idea! I like this, and we published a snapshot of `@clerk/tanstack-react-start` to NPM. We can proceed with this warning message now 🫡

Reply:
After this gets merged and published, I'll open [this PR](https://github.com/clerk/javascript/pull/5342) that will add `@clerk/tanstack-start` to ignored packages of changesets